From ac78cfaaff2f41ef2a1084c79d06ea71e858d348 Mon Sep 17 00:00:00 2001
From: zhangmeng <zhangmeng.kevin@linux.spacemit.com>
Date: Tue, 20 May 2025 22:01:14 +0800
Subject: [PATCH 31/43] riscv: config: enable spacemit p1 driver for spacemit
 k1

community inclusion
category: feature
bugzilla: https://github.com/RVCK-Project/rvck-olk/issues/27
Reference: https://gitee.com/bianbu-linux/linux-6.6/commit/85f7aea27516e3b73527247ce42947b53ce001dd

--------------------------------

Enable spacemit p1 mfd driver, and some sub-modules
in the spacemit p1 chip.

Signed-off-by: zhangmeng <zhangmeng.kevin@linux.spacemit.com>
---
 arch/riscv/configs/k1_defconfig | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/riscv/configs/k1_defconfig b/arch/riscv/configs/k1_defconfig
index 995caf1acba3c..1940cec263515 100644
--- a/arch/riscv/configs/k1_defconfig
+++ b/arch/riscv/configs/k1_defconfig
@@ -17,4 +17,10 @@ CONFIG_I2C_SPACEMIT_K1=y
 CONFIG_SPI_SPACEMIT_K1=y
 CONFIG_SPI_SPACEMIT_K1_QSPI=y
 CONFIG_PWM_PXA=m
+CONFIG_MFD_SPACEMIT_P1=m
+CONFIG_REGULATOR_SPACEMIT_P1=m
+CONFIG_INPUT_SPACEMIT_P1_PWRKEY=m
+CONFIG_PINCTRL_SPACEMIT_P1=m
+CONFIG_RTC_DRV_SPACEMIT_P1=m
+CONFIG_SPACEMIT_P1_ADC=m
 
-- 
2.51.0

